code/modules/atmospherics/machinery/pipes/multiz.dm

```diff
@@ -0,0 +1,53 @@
+/// This is an atmospherics pipe which can relay air up/down a deck.
+/obj/machinery/atmospherics/pipe/multiz
+	name = "multi deck pipe adapter"
+	desc = "An adapter which allows pipes to connect to other pipenets on different decks."
+	icon_state = "adapter-3"
+	icon = 'icons/obj/atmospherics/pipes/multiz.dmi'
+
+	dir = SOUTH
+	initialize_directions = SOUTH
+
+	layer = HIGH_OBJ_LAYER
+	device_type = UNARY
+	paintable = FALSE
+
+	construction_type = /obj/item/pipe/directional
+	pipe_state = "multiz"
+
+	var/mutable_appearance/center = null
+	var/mutable_appearance/pipe = null
+	var/obj/machinery/atmospherics/front_node = null
+
+/* We use New() instead of Initialize() because these values are used in update_icon()
+ * in the mapping subsystem init before Initialize() is called in the atoms subsystem init.
+ * This is true for the other manifolds (the 4 ways and the heat exchanges) too.
+ */
+/obj/machinery/atmospherics/pipe/multiz/New()
+	icon_state = ""
+	center = mutable_appearance(icon, "adapter_center", layer = HIGH_OBJ_LAYER)
+	pipe = mutable_appearance(icon, "pipe-[piping_layer]")
+	return ..()
+
+/obj/machinery/atmospherics/pipe/multiz/SetInitDirections()
+	initialize_directions = dir
+
+/obj/machinery/atmospherics/pipe/multiz/update_icon()
+	cut_overlays()
+	pipe.color = front_node ? front_node.pipe_color : rgb(255, 255, 255)
+	pipe.icon_state = "pipe-[piping_layer]"
+	center.pixel_x = PIPING_LAYER_P_X * (piping_layer - PIPING_LAYER_DEFAULT)
+	add_overlay(pipe)
+	add_overlay(center)
```

Comment:
I assume the parent call was being dumb?

Reply:
The problem was in `pipeline_expansion` if I remember correctly, but still, doesn't make sense for a manifold of any kind to be a subtype of straight pipes, there's not that much shared logic and all.